contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -3630,3 +3636,13 @@ bbf_check_rowcount_hook(int es_processed)
 	else
 		return false;
 }
+
+static void 
+sort_nulls_first(SortGroupClause * sortcl, bool reverse)
+{
+	if (sql_dialect == SQL_DIALECT_TSQL)
```

Comment:
To test this dialect check, could you also add a test case in PG connection?

Reply:
OK